client/handler.go

```diff
@@ -122,6 +134,70 @@ func (h *Handler) Create(w http.ResponseWriter, r *http.Request, _ httprouter.Pa
 	h.r.Writer().WriteCreated(w, r, ClientsHandlerPath+"/"+c.GetID(), &c)
 }
 
+// swagger:route POST /dyn-clients public createDynOAuth2Client
```

Comment:
OIDC specifies this as ` /connect/register` in its non-normatize examples. I think we should follow that approach

https://openid.net/specs/openid-connect-registration-1_0.html

```suggestion
// swagger:route POST /connect/register public createOAuth2ClientPublic
```